prime-router/src/main/kotlin/Element.kt

```diff
@@ -101,6 +101,11 @@ data class Element(
     // value around and into the mapper itself so the interface for the
     // mapper remains as generic as possible
     val delimiter: String? = null,
+
+    // used to be able to send blank values for fields that get validated/normalized
+    // in serializers.
+    // for instance, a badly formatted yet optional date field.
+    val nullifyValue: Boolean = false
```

Comment:
I am almost thinking that we should always nullify the values and not return an error.